### PR TITLE
Enable multiarch builds

### PR DESCRIPTION
**Pull request by carlossg**
This means using an older version of tini in Alpine, from the package
for easier installation
Move TINI_VERSION to be a build arg, as it doesn't make sense at runtime
Verify tini using gpg instead of SHA

### CODE REVIEW

**Comment by carlossg** [comment]
One of the inconveniences of not using official images @ndeloof is that these images can't be easily built

**Comment by ndeloof** [comment]
@carlossg if you refer to https://blog.docker.com/2017/09/docker-official-images-now-multi-platform/, I don't think being official image is of any help, you still need to rely on [manifest-tool](https://github.com/estesp/manifest-tool) to produce one. Just official images now do support this.

we could push such a manifest, just need to produce linux-x86/linux-arm/windows flavor of our jenkins images ... let's start by doning that :P

**Comment by carlossg** [comment]
yes you need to create the manifests but docker inc will produce all the flavors for official images

**Comment by ndeloof** [comment]
They indeed offer the adequate infra to build arm, s390 or ppc images. If we don't have, I guess none of jenkins community resources (core or plugins) have been tested on those target platforms.

Please also note official image has been deprecated, but was already not recommended since we started automatically publishing to `jenkinsci` registry one year ago

**Comment by carlossg** [comment]
@ndeloof is this ok to be merged then?

**Comment by ndeloof** [comment]
I think it's ok to merge this.
Just one thought: as docker now provide a tiny-backed init with --init option, maybe we can just get rid of this to make the docker image simple and let docker managed the PID 1 issue.

**Comment by Fantoccini** [comment]
👍 for this PR

File: Dockerfile-slim [hunk 1]
```diff
@@ -67,7 +69,7 @@ USER ${user}
 
 COPY jenkins-support /usr/local/bin/jenkins-support
 COPY jenkins.sh /usr/local/bin/jenkins.sh
-ENTRYPOINT ["/bin/tini", "--", "/usr/local/bin/jenkins.sh"]
+ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/jenkins.sh"]
```
Comment (mikemoate) [inline Dockerfile-slim]: This is a breaking change for anyone with a custom entrypoint using tini, how did this get into an LTS release?

Comment (carlossg) [reply Dockerfile-slim]: let's keep the discussion in #629

**Pull request merged**